requirements.txt

```diff
@@ -1,4 +1,8 @@
 torch
 ConfigSpace
-scikit-learn>=0.22.0,<0.23
+scikit-learn
```

Comment:
Do you think we can stick with scikit learn earlier than 0.23? There are changes in 0.23 that we haven't fully tested here not in auto-sklearn